examples/with-stomp/useStomp.js

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from 'react'
```

Comment:
Put the hooks inside a hooks folder. `with-*` signals a HOC in react.

Reply:
The `with-*` means an example of Nextjs with some library or something similar. I just removed the unneeded folder for the reason of simplicity.
@StarpTech

Reply:
@alieslamifard
```js
const useClient = () => {
  const clientRef = React.useRef()

  React.useEffect(() => {
    if (!clientRef.current) {
      clientRef.current = Stomp.client(process.env.STOMP_SERVER)
    }
  }, [])

  return clientRef.current
}
```